.codegen/service.go.tmpl

```diff
@@ -44,26 +66,39 @@ var Cmd = &cobra.Command{
 {{end}}
 // start {{.KebabName}} command
 
-{{- $useJsonForAllFields := or .IsJsonOnly (and .Request (or (not .Request.IsAllRequiredFieldsPrimitive) .Request.HasRequiredNonBodyField))  -}}
-{{- $needJsonFlag := or $useJsonForAllFields (and .Request (not .Request.IsOnlyPrimitiveFields)) -}}
-{{- if .Request}}
-var {{.CamelName}}Req {{.Service.Package.Name}}.{{.Request.PascalName}}
-{{- if $needJsonFlag}}
-var {{.CamelName}}Json flags.JsonFlag
-{{- end}}
-{{end}}
-{{if .Wait}}var {{.CamelName}}SkipWait bool
-var {{.CamelName}}Timeout time.Duration{{end}}
+// Slice with functions to override default command behavior.
+// Functions can be added from the `init()` function in manually curated files in this directory.
+var {{.CamelName}}Overrides []func(
+	*cobra.Command,
+	{{- if .Request }}
+	*{{.Service.Package.Name}}.{{.Request.PascalName}},
+	{{- end }}
```

Comment:
Maybe I'm lacking context, but why does this override depend on the request structure?

Reply:
Some overrides configure a default value in the request payload.

For example: https://github.com/databricks/cli/pull/595/files#diff-4ee345ebd7125546d56e01fed1f886213d400acb53caf49a07d7c317fd9b3abbR10

Reply:
Exactly -- some existing overrides already did just that, so this is the minimal change to port stuff over.